### DISCUSSION TITLE
Using Conventional Commits

### DISCUSSION THREAD

**Discussion by iboB** [alpaca-core/alpaca-core-sdk#125] [Dev]
[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) is a standard for commit messages. GitHub supports their tags and adds fancy boxes on commits when viewed from the browser (and I suspect the app), other git apps may also do so.

For an example see the Conventional Commits repo itself: https://github.com/conventional-commits/conventionalcommits.org/commits/master/

Shall we adopt this style?

**Comment by iboB** [comment]
Just did two commits to see how it feels. I think I like it :)

**Comment by iboB** [comment]
may be useful https://github.com/conventional-changelog/commitlint

**Comment by IvanFilipov** [comment]
I like it! :)

**Comment by zah** [comment]
FWIW, the Blocksense team is using this and most people are happy about it.

**Comment by iboB** [comment]
Accepted then. Moving on with #135
